data/Nigeria/Senate/sources/manual/terms.csv

```diff
@@ -0,0 +1,2 @@
+id,name,start_date,end_date,source
+2015,2015–,2015-06-09,,https://en.wikipedia.org/wiki/National_Assembly_(Nigeria)
```

Comment:
Per https://github.com/everypolitician/everypolitician/wiki/terms.csv it would be better to have a `wikidata` column there than a `source`. [Q28555175](https://www.wikidata.org/wiki/Q28555175) is the 8th Assembly.